Jamfile

```diff
@@ -788,13 +788,26 @@ lib torrent
 
 headers = [ path.glob-tree include/libtorrent : *.hpp ] ;
 
-package.install install
+package.install install-torrent-lib
 	: <install-source-root>libtorrent
 	:
 	: torrent
 	: $(headers)
 	;
 
+local paths = [ package.paths torrent-rasterbar ] ;
+
+local datarootdir = [ $(paths).datarootdir ] ;
+
+install install-cmake
+	: examples/cmake/FindLibtorrentRasterbar.cmake
+	: <location>$(datarootdir)/cmake/Modules
+	;
+
+alias install : install-torrent-lib install-cmake ;
+
+explicit install install-cmake ;
```

Comment:
`explicit install-cmake` alone did not work. Why?

Reply:
I suppose even an alias is implicitly built, so both targets need to be marked explicit.

Reply:
Probably. I would expect an alias to be completely transparent...